### PR TITLE
app-admin/{ansible-cmdb,clustershell}: bump

### PR DESCRIPTION
**Pull request by monsieurp**
* PYTHON_COMPAT bump (3.9)
* EAPI 8 bump

### CODE REVIEW

**Comment by monsieurp** [comment]
@arthurzam

File: app-admin/ansible-cmdb/ansible-cmdb-1.30-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_9 )
```
Comment (arthurzam) [inline app-admin/ansible-cmdb/ansible-cmdb-1.30-r1.ebuild:6]: Please don't remove python3.8 support if it works
``` suggestion
PYTHON_COMPAT=( python3_{8..9} )
```

File: app-admin/ansible-cmdb/ansible-cmdb-1.30-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+DESCRIPTION="Ansible Configuration Management Database"
+HOMEPAGE="https://github.com/fboender/ansible-cmdb"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]"
```
Comment (arthurzam) [inline app-admin/ansible-cmdb/ansible-cmdb-1.30-r1.ebuild]: setuptools can be removed from DEPEND, it is maintained correctly by the `distfiles-r1.eclass`
```suggestion
```

File: app-admin/clustershell/clustershell-1.8.3-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+DESCRIPTION="Python framework for efficient cluster administration"
+HOMEPAGE="https://cea-hpc.github.com/clustershell/"
+SRC_URI="https://github.com/cea-hpc/clustershell/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc test"
+
+CDEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
```
Comment (arthurzam) [inline app-admin/clustershell/clustershell-1.8.3-r1.ebuild]: setuptools can be removed from DEPEND, it is maintained correctly by the `distfiles-r1.eclass`
Then We can remove the `${CDEPEND}`

File: app-admin/clustershell/clustershell-1.8.3-r1.ebuild [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+DEPEND="
+	${CDEPEND}
+	test? ( dev-python/nose[${PYTHON_USEDEP}] )"
```
Comment (arthurzam) [inline app-admin/clustershell/clustershell-1.8.3-r1.ebuild]: Please put test dependencies for python world under `BDEPEND` as we support testing only on CHOST.

**Review by arthurzam — CHANGES_REQUESTED** [review]
Tested in my system and it looks like it builds. I have put some changes that improve quality of ebuild, hope it's OK :smile:

**Review by arthurzam — APPROVED** [review]
Looks great :smile:
Thank you

**Pull request closed without merging**